site/zenodo_rdm/previewer/mirador.py

```diff
@@ -19,15 +22,32 @@ def can_preview(file):
     # supported_extensions list needs . prefixed -
     preview_extensions = current_app.config["MIRADOR_PREVIEW_EXTENSIONS"]
     supported_extensions = ["." + ext for ext in preview_extensions]
-    return (
-        file.has_extensions(*supported_extensions) and file.record.data["is_published"]
-    )
+
+    return file.has_extensions(*supported_extensions)
 
 
 def preview(file):
     """Render template."""
+    media_file_name = file.filename + ".ptif"
+
+    ext = splitext(file.filename)[1].lower()[1:]
+    format = (
+        ext
+        if ext in current_app.config["IIIF_SIMPLE_PREVIEWER_NATIVE_EXTENSIONS"]
+        else "jpg"
+    )
+    size = LocalProxy(lambda: current_app.config["IIIF_SIMPLE_PREVIEWER_SIZE"])
+    url = f"{file.data['links']['iiif_base']}/full/{size}/0/default.{format}"
+
     return render_template(
         "invenio_app_rdm/records/mirador_preview.html",
+        css_bundles=["mirador-previewer.css"],
         file=file,
+        file_url=url,
+        media_file=(
+            file.record._record.media_files[media_file_name]
+            if file.record._record.media_files.enabled
+            else None
+        ),
```

Comment:
```suggestion
        media_file=(
            file.record._record.media_files[media_file_name]
            if file.record._record.media_files.enabled
            else None
        ),
```

- This might fail if the file is not there at all, in which case maybe we add either a `media_file_name in file.record._record.media_files` check or use `file.record._record.media_files.get(...)`
- I would extract this in a variable

Reply:
If we use the file.record._record.media_files.get(...) we get `{"status": 403, "message": "Files are not enabled."}` error since we're trying to access media_files when they have not been enabled

Reply:
I would keep the `file.record._record.media_files.enabled` check, but also be more "defensive" in case the file is not there (e.g. for existing Zenodo records that haven't been scheduled for tiles generation).